e2e-tests/portal/cypress/support/trade-finance-manager-api/submitDeal.js

```diff
@@ -2,4 +2,4 @@ const { submitDealAfterUkefIds } = require('./api');
 
 // This allows us to simulate a submission which would
 // normally be done after a successful scheduled Number Generator function call
```

Comment:
Number Generator no longer exists - should this command?

Reply:
`submitDealAfterUkefIds` used to be called in `number-generator.controller.js` but is now only used on the endpoint `/deals/submitDealAfterUkefIds`.
This endpoint is not called anywhere in functional code and is only used in these `submitDeal` commands. Given that this no longer accurately represents the submitting a deal process I think these functions & endpoints need removing.

Reply:
Confirmed with Abhi that this is obsolete, and raised ticket [DTFS2-7112](https://ukef-dtfs.atlassian.net/browse/DTFS2-7112). I'll add some comments to the code too